### ISSUE TITLE
Atom plugin

### ISSUE THREAD

**Issue by isakgrozny** [inkle/inky#44]
Would it be possible to make an Inky Atom plugin?

If it is, I'm willing to give it a shot myself, I'm just not quite sure if it *is* achievable.

**Comment by joethephish** [comment]
By an "inky atom plugin", do you mean an ink plugin to make atom inky-like?
:)

Yeah, should definitely be possible, and you could reuse some code from
inky. Inky is built on Electron, which is part of the Atom project (it's
the foundation). I haven't used Atom itself much though, and I have no idea
what their plugin architecture is like.

**Comment by isakgrozny** [comment]
Yup, that's what I meant! An way to preview ink games in Atom itself. I prefer using Atom to actually write the code and prose, but switching between it and Inky is a pain.

I'll take a look at how the plugin stuff works and when/if I have something workable, I'll post it here.

**Comment by joethephish** [comment]
Cool, sounds good! It's probably a fairly Major Project, depending on how full featured you want it to be, but the basics may be easy enough.

**Comment by isakgrozny** [comment]
Eventually, I'd like it to be a moderately-featured IDE, but initially I'm just going to work on the very basic "display compiled, playable ink story as I'm writing the ink source", since that's the most helpful aspect of Inky.
